supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJobCachingActorHelper.scala

```diff
@@ -56,13 +56,6 @@ trait AwsBatchJobCachingActorHelper extends StandardCachingActorHelper {
   lazy val callRootPath: Path = callPaths.callExecutionRoot
   lazy val returnCodeFilename: String = callPaths.returnCodeFilename
   // lazy val returnCodePath: Path = callPaths.returnCode
```

Comment:
More `//`s to remove?

Reply:
Sure. But I'll ask you don't go looking for more as that's a PR in and of itself 😛

Reply:
Oh wait, i intentionally left that one. I can 💯 guarantee that it'll be uncommented in the future

Reply:
And now that I'm not running to catch a bus - this is going to be uncommented when whomever implements `continueOnReturnCode` implements `continueOnReturnCode`. I think it's better to leave those sorts of breadcrumbs here as it makes it easier for dopes like myself to follow the trail instead of figure it out from scratch :)

Reply:
Personally every time I've left something like that for future-Chris, future-Chris ended up reimplementing the same line elsewhere anyway (or sometimes even right next to the comment without noticing).

But 🤷‍♂️ I don't want to hold up the PR over a preexisting commented line.